core/src/main/java/dev/geco/gmusic/manager/SongManager.java

```diff
@@ -2,20 +2,24 @@
 
 import java.io.*;
 import java.util.*;
+import java.util.stream.Collectors;
 
+import net.kyori.adventure.text.Component;
```

Comment:
I'm not sure how to get rid of the errors on strings vs. components since you're using both spigot and paper as a dependency - in my IntelliJ environment, paper seems to take priority over compile time. I have buildtools for 1.21 installed.